guides/swoosh_mailer.md

```diff
@@ -22,8 +24,15 @@ defmodule MyAppWeb.PowMailer do
 
   def process(email) do
     deliver(email)
```

Comment:
This can be written:

```elixir
email
|> deliver()
|> log_warnings()
```

I expand pipes completely so the first input is a variable.

Reply:
Thanks, this is a bad habit that I need to break.